src/main/kotlin/com/github/ast/parser/NodeExtensions.kt

```diff
@@ -0,0 +1,62 @@
+package com.github.ast.parser
+
+import com.google.gson.JsonArray
+import com.google.gson.JsonObject
+
+fun JsonObject.getType() = this.pieces().getObject(0).name()
```

Comment:
Moving these into a separate file is great! Really helps clean up the KParser.

Reply:
Fr, as if that file isn't large enough already. Also, it works well as a set of independent functions!